app/Http/Middleware/CheckAdmin.php

```diff
@@ -16,8 +16,10 @@ class CheckAdmin
      */
     public function handle($request, Closure $next)
     {
-        if (Auth::user()->role == config('site.permission.admin') || Auth::user()->role == config('site.permission.trainer'))
-        {
+        if (
+            Auth::user()->role == config('site.permission.admin') 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Http/Middleware/CheckAdmin.php</br>Whitespace found at end of line